KenticoOnboardingApplication/KenticoOnboardingApplication.Api.Tests/ListControllerTest.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using KenticoOnboardingApplication.Api.Controllers;
+using NUnit.Framework;
+using System.Web.Http;
+using KenticoOnboardingApplication.Api.Models;
+using KenticoOnboardingApplication.Api.Tests.Comparers;
+
+namespace KenticoOnboardingApplication.Api.Tests
+{
+    [TestFixture]
+    public class ListControllerTest
+    {
+        private ListController _controller;
+        private static readonly Guid Guid = new Guid("d95f4249-6f37-46ab-b102-b55972306910");
```

Comment:
You really don't need these fields (Guid, Item) as you have whole array of Items that should be used.